README.md

```diff
@@ -328,7 +328,7 @@ to _not_ have to type all that each time. e.g:
 
 ```js
 "scripts": {
-  "e2e": "node_modules/.bin/nightwatch --config nightwatch.conf.BASIC.js"
+  "e2e": "nightwatch --config nightwatch.conf.BASIC.js"
```

Comment:
we prefer the _full_ path to make it clear that the module is installed _locally_, but this will work so I don't mind merging it. 👍